generators/java-v2/sdk/src/SdkGeneratorContext.ts

```diff
@@ -26,4 +38,215 @@ export class SdkGeneratorContext extends AbstractJavaGeneratorContext<SdkCustomC
             readmeConfigBuilder: new ReadmeConfigBuilder()
         });
     }
+
+    public getReturnTypeForEndpoint(httpEndpoint: HttpEndpoint): java.Type {
+        const responseBody = httpEndpoint.response?.body;
+
+        if (responseBody == null) {
+            return java.Type.void();
```

Comment:
Ah I see now how you're using `Void` - if we need the capitlaized equivalent, thoughts on adding an optional parameter here? Something like the following:

```
java.Type.void() // Produces `void` for method signatures

java.Type.void({ captialized: true }) // Produces `Void` for variable type references.
```